### DISCUSSION TITLE
发送消息失败：blocked by server

### DISCUSSION THREAD

**Discussion by JohnnieWind** [Mrs4s/go-cqhttp#2221] [Q&A]
今天发现发送消息失败了，是被风控了吗？怎么解决
{data=null, message=send group message failed: blocked by server, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=send group message failed: blocked by server}

**Comment by Dragon-ts** [comment]
可以试试删掉token和device.json
